config-stub.backend.yml

```diff
@@ -0,0 +1,11 @@
+github-user: XXX
+github-password: XXX
+jira-server: https://tangle.atlassian.net
+jira-user: XXX
+jira-password: XXX
+jira-project: BAC
+jira-release-project: RELEASE
+origin-branch: master
+jira-task-extra:
+  component: Backend
+release-set: yarn version --no-git-tag-version --new-version
```

Comment:
It's better to set `echo` here, see my comment. Later we are going to change it.

Reply:
Fixed.